src/components/ExportButton.js

```diff
@@ -1,17 +1,20 @@
 import React from "react";
 import {downloadFromText} from "../utils/download";
-import {Button} from "antd";
 import csvStringify from "csv-stringify/lib/sync";
+
+import {Button} from "antd";
+import "antd/es/button/style/css";
+
 import { DownloadOutlined } from "@ant-design/icons";
 
 
-const ExportButton = ({ exportFunction, fileName }) => {
-  var fileName = fileName + '_' + (new Date()).toJSON() + '.csv'
+const ExportButton = ({ exportFunction, filename }) => {
+  let name = filename + '_' + new Date().toISOString().slice(0,10) + '.csv'
```

Comment:
Space after comma. I don't know which editor you use but you should set it up to catch common linting errors. I prefer to keep the code as clean & consistent as possible for readability reasons.